src/java/za/co/mip/ablduck/javadoc/JavadocParser.java

```diff
@@ -0,0 +1,1797 @@
+// Generated from /home/roberte/Projects/pct/src/java/za/co/mip/ablduck/javadoc/JavadocParser.g4 by ANTLR 4.7
+package za.co.mip.ablduck;
+import org.antlr.v4.runtime.atn.*;
+import org.antlr.v4.runtime.dfa.DFA;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.misc.*;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Explicitly import the specific classes needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2208)